### ISSUE TITLE
More detail for processes, organisation and infrastructure - by expanding the CRE structure beyond technical application security

### ISSUE THREAD

**Issue by robvanderveer** [OWASP/common-requirement-enumeration#250]
Background
OpenCRE was created by application security professionals from a developer perspective, and therefore the first adopted standards were appsec standards and the higher level structure was based on appsec - primarily using SIG’s ISO25010-based model of security as a software quality.

The issue
As more standards were added to CRE, some topics in those standards didn’t really fit where it was outside of software, for example network segregation (network security) but also documentation of assets and their risks (process) and employee security awareness (organisation). Also hardware security didn’t really have a place (e.g. removing descriptions from printed circuit boards). The latter can be compared to removing comments from Javascript code, but it’s a separate countermeasure.

Especially with engineering teams becoming increasingly involved with process improvement and operations, the CRE team believes CRE should go beyond application security in order to fulfill its primary use case: enable software security professionals to find relevant information and the other way around: to allow information providers to deliver it effectively.
By being more comprehensive, it can become clearer for software security professionals that there are countermeasures to certain threats beyond software - for example, apart from protecting an isolated network in a room, one could in addition put an armed guard in front of the door.
The nice side-effect of this scope-increase is that CRE becomes available for more use cases to a larger audience: people who run organisation security programs, people who are more hardware-involved, etc.

In other words: OpenCRE needs to go beyond application security to cover all security measures in order to help software security professionals, and by doing so, this will help other security professionals as well.

The solution
So, OpenCRE needs a more comprehensive structure to hold security topics.

**Comment by robvanderveer** [comment]
This work is finally done!
